ansible/roles/kamailio/tasks/main.yml

```diff
@@ -0,0 +1,78 @@
+---
+- name: Install gnupg2
+  become: true
+  apt:
```

Comment:
As we work with Ansible >= 2.10.0, the FQCN of the action of every task should be written (`ansible.builtin.apt` instead of `apt`)